### PR TITLE
[RF] POM |AT_01.01 _021| Verify Head Icon is clickable.

### PR DESCRIPTION
**Pull request by KseniyaParfianovich**
https://trello.com/c/mDlLrZag/1576-rf-pom-at0101-021-verify-head-icon-is-clickable

### CODE REVIEW

**Review by SawkaQA — APPROVED** [review]

File: cypress/pageObjects/HomePage.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import MultiConfigurationProjectPage from "./MultiConfigurationProjectPage";
 
 class HomePage {
     getPeopleSideMenuLink = () => cy.get('a[href="/asynchPeople/"]');
+    getHomepageHeader = () => cy.get('.empty-state-block h1');
```
Comment (IrynaKolh) [inline cypress/pageObjects/HomePage.js:8]: Move line 8 before line 7. Shouldn't split SideMenuLinks. Must be understanding order of css selectors.

File: cypress/pageObjects/UserConfigurePage.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import UserProfilePage from "../pageObjects/UserProfilePage"
+import HomePage from "../pageObjects/HomePage";
 
 class UserConfigurePage {
     getFullNameInputField = () => cy.get('input[name="_.fullName"]');
     getUserConfigSaveBtn = () => cy.get('button[name="Submit"]');
+    getJenkinsHeaderIcon = () => cy.get('#jenkins-head-icon');
```
Comment (IrynaKolh) [inline cypress/pageObjects/UserConfigurePage.js:7]: This css celector doesn't belong to UserConfigurePage. It belongs to HeaderAndFooter. HeaderAndFooter presents in any pages, so move this css celector and clickJenkinsHeaderIcon() method to class HeaderAndFooter

File: cypress/pageObjects/UserConfigurePage.js [hunk 3]
```diff
@@ -17,5 +19,9 @@ class UserConfigurePage {
         return new UserProfilePage();
     }
 
+    clickJenkinsHeaderIcon() {
```
Comment (IrynaKolh) [inline cypress/pageObjects/UserConfigurePage.js:22]: move this  clickJenkinsHeaderIcon() method to class HeaderAndFooter

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 4]
```diff
@@ -13,6 +14,15 @@ describe('headerAndFooter', () => {
...
         .should('have.text',restAPIPageTitle)
+    })
+
+    it('AT_01.01 _021| Verify Head Icon is clickable.', () => {
+        headerAndFooter
+        .clickUserDropDownBtn()
+        .selectUserConfigureMenu()
+        .clickJenkinsHeaderIcon()
+        .getHomepageHeader()
+        .should('have.text', homePageHeader);
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js]: Please add 4 spaces/1 tab for methods

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 5]
```diff
@@ -14,6 +15,16 @@ describe('headerAndFooter', () => {
         .clickRestAPILink()
         .getRestApiTitle()
         .should('have.text',restAPIPageTitle)
+    })
+
+    it('AT_01.01 _021| Verify Head Icon is clickable.', () => {
+        headerAndFooter
+        .clickUserDropDownBtn()
+        .selectUserConfigureMenu()
+        headerAndFooter
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:24]: Preferably add an empty line after the chain is broken

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 6]
```diff
@@ -14,6 +15,16 @@ describe('headerAndFooter', () => {
         .clickRestAPILink()
         .getRestApiTitle()
         .should('have.text',restAPIPageTitle)
+    })
+
+    it('AT_01.01 _021| Verify Head Icon is clickable.', () => {
+        headerAndFooter
+        .clickUserDropDownBtn()
+        .selectUserConfigureMenu()
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:23]: Please add `;` to end line to show that the chain is over

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 7]
```diff
@@ -2,6 +2,7 @@
 
 import HeaderAndFooter from "../../pageObjects/HeaderAndFooter";
 import {restAPIPageTitle} from "../../fixtures/pom_fixtures/restAPIPage.json"
+import{homePageHeader} from "../../fixtures/pom_fixtures/homePage.json";
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:5]: add spaces  `import {homePageHeader}`

File: cypress/pageObjects/HeaderAndFooter.js [hunk 8]
```diff
@@ -41,6 +42,10 @@ class HeaderAndFooter {
         return new HomePage();
     }
 
+    clickJenkinsHeaderIcon() {
+        this.getJenkinsHeaderIcon().click();
+        return new HomePage();
+   }
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:48]: please delete as it's already created `clickJenkinsHomeLink()`. Use it

File: cypress/pageObjects/HeaderAndFooter.js [hunk 9]
```diff
@@ -10,6 +10,7 @@ class HeaderAndFooter {
     getUserDropdownMenuItemsList = () => cy.get('.bd li');
     getRestAPILink = () => cy.get('[href="api/"]');
     getJenkinsHomeLink = () => cy.get('#jenkins-home-link');
+    getJenkinsHeaderIcon = () => cy.get('#jenkins-head-icon');
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:13]: please delete as it's already created `getJenkinsHomeLink`

Comment (KseniyaParfianovich) [reply cypress/pageObjects/HeaderAndFooter.js]: Thank you for review. I've fixed all comments and skipped some of the same tests. What do you say?

**Review by Rom-T — APPROVED** [review]

**Pull request merged**